README.md

```diff
@@ -1 +1,18 @@
-# beyond08-group1
\ No newline at end of file
+# Tasties (beyond08-group1) 
+
+### Introduction :
+Welcome to our project Tasties!!
+
+
+### Team members: 
+Gal Oved , Waseem Abu Sobe , Efraim Vishnivetzky , Ohad Revah , Gil Shechter
+
+
+### Idea :  
+The idea of the project
```

Comment:
Maybe refer to it as the `Project` and not the `Idea` since you're past the phase of ideas already and working on a project